src/praisonai-agents/praisonaiagents/tools/schedule_tools.py

```
@@ -155,7 +158,23 @@ def schedule_add(
-        return f"Schedule '{name}' added (id: {job.id}, {schedule}{agent_note}{delivery_note})."
+        confirmation = f"Schedule '{name}' added (id: {job.id}, {schedule}{agent_note}{delivery_note})."
+
+        # ── Suggestion acceptance integration ─────────────────────
+        if accept_suggestion:
+            try:
+                from ..scheduler.suggestion_store import SuggestionStore
+                store_obj = SuggestionStore()
+                sug = store_obj.get(accept_suggestion)
+                if sug is not None and not sug.accepted and not sug.dismissed:
+                    store_obj.accept(accept_suggestion)
+            except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)